### PR TITLE
Resolve ambiguity between divexact methods

### PR DESCRIPTION
**Pull request by fingolfin**

### CODE REVIEW

**Comment by thofma** [comment]
Do you have an example? Or was this reported from JET?

**Comment by fingolfin** [comment]
This was reported by JET, but you are right, an example (added as a test case) would be good. I'll look into it.

**Comment by fingolfin** [comment]
Err, Aqua reported it, not JET

File: test/generic/FunctionField-test.jl [hunk 1]
```diff
@@ -603,6 +603,11 @@ end
       # characteristic 0
       S, y = function_field(P1[i], "y")
 
+      @test divexact(2, y) * y == 2
+      @test divexact(big(2), y) * y == 2
+      @test divexact(1//2, y) * y == 1//2
+      @test divexact(big(1)//2, y) * y == 1//2
```
Comment (fingolfin) [inline test/generic/FunctionField-test.jl:609]: This last new test results in an ambiguity error on master, and is fixed by this PR.

**Comment by thofma** [comment]
Thanks

**Pull request merged**